src/test/java/teammates/test/cases/action/BaseActionTest.java

```diff
@@ -731,4 +731,8 @@ protected void verifyNumberOfEmailsSent(Action action, int emailCount) {
         assertEquals(emailCount, action.getEmailSender().getEmailsSent().size());
     }
 
+    protected String getPageResultDestination(String viewUri, boolean error, String userId) {
+        return viewUri + "?error=" + error + "&user=" + userId;
```

Comment:
@sujeet14108 Sorry, I did not notice the sanitization used. In that case, maybe you could write a similar method to addParamToUrl in BaseActionTest to do the string processing.